website/docs/d/list_profiles.html.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_list_profiles"
+description: |-
+  Get information about list_profiles
+subcategory: "Security and Compliance Center (SCC)"
```

Comment:
This should be just Security and Compliance Center based on new comments from our doc team.

Reply:
Please check other places also